.circleci/config.yml

```diff
@@ -114,11 +114,3 @@ workflows:
           requires:
           - go-build
           - push-chart-operator-to-default-app-catalog
-      
-      - architect/integration-test:
-          context: architect
-          name: migration-integration-test
```

Comment:
Since we could not ensure `chartconfig` CRD using k8sclient/v4, we are deleting this integration test.

Reply:
This migration is nearly done so I think this is fine.